### DISCUSSION TITLE
Windows 2022 image version value

### DISCUSSION THREAD

**Discussion by retirem** [actions/virtual-environments#7100] [Image configuration]
Hi!

Is it by purpose, that in the commit that is marked as Windows 2022 latest, in the windows2022.json file the image_version value is set to **dev**?

Like this: "image_version": "dev"

Should not it be updated by any commit that is marked as latest or release? Or is it the responsibility of the one that creates the image with the help of the runner-images repository?

**Comment by erik-bershel** [comment]
A current state of the repository's main branch is not the same as the latest published runner image for any of the operating systems. Between any two deployments changes happening with our repo and with external repos too. Thus, it will not be fair enough to set the value of this label in the same way as the current latest release.

**Reply by retirem** [reply]
I meant it can be set for the release commit only and after that set it back to dev until the next release.

**Reply by erik-bershel** [reply]
And in result it wouldn't be the same image but with a same label. ImageLabel with any of versions is in use only for released runner images. All else runners will be more or less different than published one. And it is not only about changes history for our repo. It is about changes in all else repositories that used when software installing in process of build. So it still not so usable in that meaning.
